content/project/hal.md

```diff
@@ -3,6 +3,8 @@ title: Łazik HAL-062
 url: "/hal"
 main_project_image: "/images/hal_small.jpg"
 article_image: "/images/lazik/4.jpg"
+start_date:
+end_date:
 short_description:
```

Comment:
Czy musiałeś tutaj dodawać te wartości? 

Wydaje mi się, że wszystko powinno działać, nawet, gdybyś tutaj nie zdefiniował tych wartości.

Reply:
To dobrze, po prostu zacząłem się zastanawiać, czy nie dodałeś tego dlatego, że musiałeś, a nie chciałeś.